opencog/attention/AttentionParamQuery.cc

```diff
@@ -104,7 +104,7 @@ HandleSeq AttentionParamQuery::get_params(void)
 void AttentionParamQuery::load_default_values(void)
 {
      SchemeEval scm(_as);
-     scm.eval("(load \"opencog/attention/default-param-values.scm\")");
+     scm.eval("(load \"" PROJECT_BINARY_DIR "/opencog/attention/default-param-values.scm\")");
```

Comment:
Got it, should be fixed in https://github.com/opencog/opencog/pull/2612, thanks!